tools/deploy

```diff
@@ -38,28 +38,72 @@ upload_chrome_prod_ext() {
     --refresh-token $CHROME_WEBSTORE_REFRESH_TOKEN
 }
 
-sign_firefox_ext() {
+# Upload a Firefox extension to addons.mozilla.org for automated validation and
+# signing. Write log output to `ext_source_dir/log.txt`.
+#
+# Arguments:
+#   ext_id - ID of the Firefox extension
+#   ext_source_dir - Directory containing source files for the extension.
+# Returns:
+#   0 if extension was successfully signed and downloaded to `ext_source_dir`.
```

Comment:
nb. The syntax for documenting arguments and return types comes from Google's [shell script style guide](https://google.github.io/styleguide/shellguide.html).

Reply:
Probably not in _this_ script as it would be more of an organization-wide convention.